api/src/certification/results/domain/services/session-results-link-service.js

```diff
@@ -0,0 +1,8 @@
+import { config } from '../../../../shared/config.js';
+import { tokenService } from '../../../../shared/domain/services/token-service.js';
+
+export const generateResultsLink = function ({ sessionId, i18n }) {
+  const token = tokenService.createCertificationResultsLinkToken({ sessionId });
+  const lang = i18n.getLocale();
+  return `${config.domain.pixApp + config.domain.tldOrg}/resultats-session#${token}?lang=${lang}`;
```

Comment:
Cette URL est fausse, on a inversé l'anchor et les query.
Ce devrait être 
```suggestion
  return `${config.domain.pixApp + config.domain.tldOrg}/resultats-session?lang=${lang}#${token}`;
```
Cela explique pourquoi côté front on a également le token quand on lit la proprité `location.hash`.